x-pack/plugins/security_solution/server/lib/detection_engine/routes/rules/create_rules_route.ts

```diff
@@ -94,7 +95,9 @@ export const createRulesRoute = (router: IRouter, ml: SetupPlugins['ml']): void
         const query = !isMlRule(type) && queryOrUndefined == null ? '' : queryOrUndefined;
 
         const language =
-          !isMlRule(type) && languageOrUndefined == null ? 'kuery' : languageOrUndefined;
+          !isMlRule(type) && type !== 'eql' && languageOrUndefined == null
```

Comment:
optional: You could pull in the `isEqlRule` from the common file you have defined here if you want I thnk:

```ts
x-pack/plugins/security_solution/common/detection_engine/utils.ts
```

Feels weird that the `isMlRule(type)` is in its own helper rather than with the rest. We probably would move it over so they're all together in the same file of that utils.

Reply:
Good point, updated to use `isEqlRule` here and in `signal_rule_alert_type`. Agreed on unifying the utils - probably in a PR which doesn't attempt to add any functionality.